package.json

```diff
@@ -16,7 +16,7 @@
     "semantic-release": "semantic-release pre && npm publish && semantic-release post"
   },
   "lint-staged": {
-    "*.js": [
+    "{bin,config,scripts}/**/*.js": [
```

Comment:
Just leave it for now. Really want to get this merged. We can fix the prettier config later if it becomes an issue.